src/components/Icon.vue

```diff
@@ -18,4 +18,7 @@ export default {
   line-height: 16px;
   vertical-align: baseline;
 }
+.icon--red {
+  color: red !important;
```

Comment:
We don't need to create a new class for it, you can use `text-red` from PrimerCSS. This can be passed with `class="text-red"` we don't need a props within Icon component

Reply:
Doesn't work with warning icon for some reason, any idea why? Same for the info icon.

Reply:
The class `text-red` should use important, maybe that's because there is a `text-gray` class that overwrite the text-red

Reply:
Yes, that was it. Pushed the changes